### PR TITLE
chore: add note on relation fields reintrospection

### LINKED ISSUE #727: Link to Re-Introspection documentation in "Renaming Relation Fields"
https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/configuring-the-prisma-client-api#renaming-relation-fields

Context: https://twitter.com/i/web/status/1289660815549718530

### PR DESCRIPTION
**Pull request by ruheni**
Resolves #727

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx [hunk 1]
```diff
@@ -302,3 +302,9 @@ model User {
   favoritedPosts Post[] @relation("Post_favoritedByToUser")
 }
 ```
+
+<Admonition>
+
+Prisma `db pull` preserves your relation fields in your Prisma schema on re-introspection of your database.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx]: Is "Re-introspection" a public term we use?
What does "preserves your relation fields" mean specifically?

Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx]: > Is "Re-introspection" a public term we use?

I'm not sure, but I've seen it used in use in a few other pages in the docs. 
- https://www.prisma.io/docs/concepts/components/prisma-schema/views#adding-a-unique-identifier-to-an-introspected-view:~:text=When%20re%2Dintrospecting%20your%20database%2C%20any%20custom%20changes%20to%20your%20view%20definitions%20will%20be%20preserved.
- https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps-typescript-postgres#change-the-database-schema-eg-add-more-tables:~:text=Re%2Dintrospect%20your%20database

> What does "preserves your relation fields" mean specifically?

It should mean that any custom relation names be kept as they are without modification (by Prisma). I'll revise this.

Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx]:
```suggestion
Prisma `db pull` preserves custom relation fields defined in your Prisma schema on re-introspecting your database.
```

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx [hunk 2]
```diff
@@ -302,3 +302,9 @@ model User {
   favoritedPosts Post[] @relation("Post_favoritedByToUser")
 }
 ```
+
+<Admonition>
+
+Prisma `db pull` preserves custom relation fields defined in your Prisma schema on re-introspecting your database.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx]: Either "Prisma's `db pull`" or "`prisma db pull`" usually I think.

Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx]:
```suggestion
`prisma db pull` preserves custom relation fields defined in your Prisma schema on re-introspecting your database.
```

**Review by janpio — APPROVED** [review]

**Pull request merged**